### DISCUSSION TITLE
Auto complete

### DISCUSSION THREAD

**Discussion by 0xDTC** [TriliumNext/Trilium#2968] [Ideas]
I'm looking for autocomplete option if possible to make it'll make things fast like creating notes getting code extra.

**Comment by zadam** [comment]
Hi, you mean autocomplete within (text/code...) notes, right? That is not available ...

**Reply by 0xDTC** [reply]
Yes i know but can't you add this functionality in it

**Comment by andersmmg** [comment]
I'd love something like / 'command' autocomplete, for quickly doing things you'd normally do with the paragraph button on lines. Stuff like inserting notes, adding types of text blocks, etc.

**Comment by mike1936** [comment]
I have found this npm package from CodeMirror which might be relevant to this idea
https://github.com/codemirror/autocomplete

**Reply by 0xDTC** [reply]
Not in code only, but in text also code can be copy paste from IDE. But text like writing note, writeups, blog, report etc auto-complete can make that easy and write fast

**Reply by 0xDTC** [reply]
and if possible please can you guys include mind-map feature also

i have so-many ideas which can make this APP god😶‍🌫️ like APP but i'm not a developer😭
